3469/1/html_parser.rb

```diff
@@ -0,0 +1,38 @@
+class HtmlParser
+  attr_reader :gem_information
+  def initialize(full_name)
```

Comment:
It is bad practice to return value from initialize, you should add another public method